### PR TITLE
Feature/fastify

### PR DESCRIPTION
**Pull request by ivanjj22**
add: stack fastify

### CODE REVIEW

File: nodejs/fastify/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM node:15.8.0-alpine3.13
+ENV NODE_ENV production
+WORKDIR /app/commons
+COPY commons .
+RUN npm install --only=production
```
Comment (juancgalvis) [inline nodejs/fastify/Dockerfile:5]: Docker image is not running successfully

Error:
```
> nodejs-fastify@1.0.0 start
> node index.js

setting up routes /app/fastify/routes/crypt.js
node:internal/modules/cjs/loader:1167
  return process.dlopen(module, path.toNamespacedPath(filename));
                 ^

Error: Error loading shared library /app/commons/node_modules/bcrypt/lib/binding/napi-v3/bcrypt_lib.node: Exec format error
    at Object.Module._extensions..node (node:internal/modules/cjs/loader:1167:18)
    at Module.load (node:internal/modules/cjs/loader:973:32)
    at Function.Module._load (node:internal/modules/cjs/loader:813:14)
    at Module.require (node:internal/modules/cjs/loader:997:19)
    at require (node:internal/modules/cjs/helpers:92:18)
    at Object.<anonymous> (/app/commons/node_modules/bcrypt/bcrypt.js:6:16)
    at Module._compile (node:internal/modules/cjs/loader:1108:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)
    at Module.load (node:internal/modules/cjs/loader:973:32)
    at Function.Module._load (node:internal/modules/cjs/loader:813:14) {
  code: 'ERR_DLOPEN_FAILED'
}
npm ERR! code 1
npm ERR! path /app/fastify
npm ERR! command failed
npm ERR! command sh -c node index.js
```

**Review by juancgalvis — CHANGES_REQUESTED** [review]
Please review comments

File: nodejs/fastify/Dockerfile (hunk 1 above)
Comment (ivanjj22) [reply nodejs/fastify/Dockerfile:5]: Hi @juancgalvis, it seems like docker is copying the node_modules folder, the solution is adding the .dockerignore file and specify the value `**/node_modules`.

I've added that file and it should work.

image

image

image

image

**Review by juancgalvis — APPROVED** [review]
thanks for your contribution

**Pull request merged**